jena-arq/src/main/java/org/apache/jena/sparql/core/DynamicDatasets.java

```diff
@@ -56,8 +55,8 @@ public static DatasetGraph dynamicDataset(DatasetDescription description, Datase
     	if (description.isEmpty() )
     		return dsg;
     	
-        Set<Node> defaultGraphs = NodeUtils.convertToNodes(description.getDefaultGraphURIs()) ; 
-        Set<Node> namedGraphs = NodeUtils.convertToNodes(description.getNamedGraphURIs()) ;
+        Collection<Node> defaultGraphs = NodeUtils.convertToNodes(description.getDefaultGraphURIs()) ; 
```

Comment:
Isn't the point of using sets here to remove duplicates or is that now done in the `NodeUtils.convertToNodes()` method?

Reply:
There are various places duplicate removal may happen but it is confusing and with all the routes may be depending on here. Better to maintain compatibility.

PR #464 restores the explicit behaviour by having "to list" and "to set" versions and a deprecated "NodeUtils.convertToNodes" with the Set semantics.

Does that look good?